app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,8 @@
 class PostsController < ApplicationController
   def index
-    @posts = User.find(params[:user_id]).posts.order(created_at: :desc)
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.order(created_at: :desc)
+    @comments = Comment.includes(:user).where(post_id: @posts.ids).order(created_at: :desc)
```

Comment:
- Please ensure your app solves the `N+1` problem when fetching all posts and their comments for a user as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLAZ4K2ZHA777536NECDBHORQS) 
E.g: in the posts#index actions, you should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts.order`